app/javascript/src/styles/specific/comments.scss

```diff
@@ -1,6 +1,12 @@
 @import "../base/000_vars.scss";
 
 div.comments-for-post {
+  div.moderation-comments-notice {
+    margin: 1em 0;
+    font-weight: bold;
+    color: red;
```

Comment:
CSS colors should use variables defined in `app/javascripts/src/styles/base/040_colors.css` instead of being hardcoded. This is so that light and dark mode can use the right colors.